SPCTR-3026: Fix - Advance Heading line height issue in frontend with astra theme. (#3324)
brainstormforce/ultimate-addons-for-gutenberg: README.md

* Fix: Image - Resolved an issue in which the tablet and mobile border css would not be applied as intended.
* Fix: Forms - Reworked the e-mail sending functionality to ensure that only one e-mail is sent when CC and BCC addresses are specified.
* Fix: Icon - Resolved an issue in which the icon's background and border color on hover would not work as intended.

> * Fix: Advance Heading - Resolved an issue in which line-height is not same as editor when astra theme is activated.

Reviewer: ```suggestion
* Fix: Heading - Resolved an issue in which the line-height would be applied incorrectly in the editor when using the Astra theme.
```

Author: Done.
